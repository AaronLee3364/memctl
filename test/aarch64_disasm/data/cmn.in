fffffff0070c0000 b100069f 2b09011f ab39d17f
