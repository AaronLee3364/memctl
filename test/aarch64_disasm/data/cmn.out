fffffff0070c0000  CMN     X20, #0x1
fffffff0070c0004  CMN     W8, W9
fffffff0070c0008  CMN     X11, W25, SXTW #4
